qemu/tests/cfg/cpu_device_hotplug_during_boot.cfg

```diff
@@ -0,0 +1,12 @@
+- cpu_device_hotplug_during_boot: cpu_device_hotpluggable
+    required_qemu = [2.6.0, )
```

Comment:
seems to be a typo?

Reply:
AFAIK, qemu supports vCPU hotplug from 2.6.0, please correct it if I am wrong.

Reply:
I meant syntax `[2.6.0, )` ?

Reply:
Oh, it's not a typo, here is the usage.
https://github.com/avocado-framework/avocado-vt/blob/85d0e787d7994416bf77ce2aad8b5ef3061df4dd/virttest/utils_version.py#L6-L23